app/api/__init__.py

```diff
@@ -69,3 +73,9 @@
 api.route(TrackDetail, 'track_detail', '/tracks/<int:id>', '/sessions/<int:session_id>/track')
 api.route(TrackRelationship, 'track_sessions', '/tracks/<int:id>/relationships/sessions')
 api.route(TrackRelationship, 'track_event', '/tracks/<int:id>/relationships/event')
+
+# session_types
+api.route(SessionTypeList, 'session_type_list', '/session_types', '/events/<int:event_id>/session_types')
```

Comment:
@shubham-padia this should exist only as a relationship to an event. `/session_types` shouldn't exist